### ISSUE TITLE
Disable email field on Manage Account page when user has authenticated via AAF

### ISSUE THREAD

**Issue by carlohamalainen** [mytardis/mytardis#333]
The user should not be able to able to edit their email address if they have authenticated via AAF as their email is their fixed username.

I can attend to this once the current AAF PR is merged.

**Comment by grischa** [comment]
Hi @carlohamalainen ,
I'm not sure about this issue, we don't use AAF anywhere here yet. Is there anything that needs to be done? Otherwise I'll just close it

**Comment by carlohamalainen** [comment]
I've written code to handle this case. Must do a new pull request on the latest stuff, might talk tomorrow at the meeting about this. So can close this issue.

**Comment by grischa** [comment]
Ok, great. Thanks
